.buildkite/bk.fips-integration.pipeline.yml

```diff
@@ -0,0 +1,169 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
+  VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
+  ASDF_MAGE_VERSION: 1.14.0
+  CUSTOM_IMAGE_TAG: "git-${BUILDKITE_COMMIT:0:12}"
+  CI_ELASTIC_AGENT_DOCKER_IMAGE: "docker.elastic.co/beats-ci/elastic-agent-fips-cloud"
+  TF_VAR_integration_server_docker_image: "${CI_ELASTIC_AGENT_DOCKER_IMAGE}:${CUSTOM_IMAGE_TAG}"
+
+  IMAGE_UBUNTU_2404_X86_64: "platform-ingest-elastic-agent-ubuntu-2404-1744855248"
+  IMAGE_UBUNTU_X86_64_FIPS: "test-platform-ingest-elastic-agent-ubuntu-2204-fips-1748876280"
+  IMAGE_UBUNTU_ARM64_FIPS: "test-platform-ingest-elastic-agent-ubuntu-2204-fips-aarch64-1748876280"
+
+steps:
+  - label: Build and push custom elastic-agent image
+    depends_on:
+      - 'packaging-containers-x86-64-fips'
```

Comment:
I cannot find where this dependency comes from. Is it expected in another BK pipeline? If so, I wonder whether we could add some context here about where that pipeline step is actually defined and the other way around, create a comment in the pipeline step `packaging-containers-x86-64-fips` to link to this BK pipeline.

Otherwise, things can be a bit complicated if no clear indication about those dependencies.

Reply:
It's from `.buildkite/integration.pipeline.yml` which calls this pipeline and `bk.integrattion.pipeline.yml`, i'll add a comment to point the source out